modules/agent-install-sample-config-bond-sriov.adoc

```diff
@@ -0,0 +1,122 @@
+     - name: eno1
+       macAddress: 0c:42:a1:55:f3:06
+     - name: eno2
+       macAddress: 0c:42:a1:55:f3:07
+    networkConfig: <1>
+      interfaces: <2>
+        - name: eno1     <3>
+          type: ethernet <4>
+          state: up
+          mac-address: 0c:42:a1:55:f3:06
+          ipv4:
+            enabled: true
+            dhcp: false  <5>
+          ethernet:
+            sr-iov:
+              total-vfs: 2 <6>
+          ipv6:
+            enabled: false
+        - name: sriov:eno1:0
+          type: ethernet
+          state: up  <7>
+          ipv4:
+            enabled: false <8>
+          ipv6:
+            enabled: false
+        - name: sriov:eno1:1
+          type: ethernet
+          state: down
+      - name: eno2
+        type: ethernet
+        state: up
+        mac-address: 0c:42:a1:55:f3:07
+        ipv4:
+          enabled: true
+        ethernet:
+          sr-iov:
+            total-vfs: 2
+        ipv6:
+          enabled: false
+      - name: sriov:eno2:0
+        type: ethernet
+        state: up
+        ipv4:
+          enabled: false
+        ipv6:
+          enabled: false
+      - name: sriov:eno2:1
+        type: ethernet
+        state: down
+      - name: bond0
+        type: bond
+        state: up
+        min-tx-rate: 100 <9>
+        max-tx-rate: 200 <10>
+        link-aggregation:
+          mode: active-backup <11>
+          options:
+            primary: sriov:eno1:0 <12>
+          port:
+            - sriov:eno1:0
+            - sriov:eno2:0
+          ipv4:
+            address:
+            - ip: 10.19.16.57 <13>
+              prefix-length: 23
+            dhcp: false
+            enabled: true
+          ipv6:
+            enabled: false
+          dns-resolver:
+            config:
+              server:
+               - 10.11.5.160
+               - 10.2.70.215
+          routes:
+            config:
+              - destination: 0.0.0.0/0
+                next-hop-address: 10.19.17.254
+                next-hop-interface: bond0 <14>
+                table-id: 254
+----
+<1> The networkConfig field holds the network configuration of the host. It has several subfields, including `interfaces`, `dns-resolver`, and `routes`.
+<2> The `interfaces` field is an array of network interfaces defined for the host.
+<3> Name of the interface.
+<4> The type of interface. This example creates a ethernet interface.
+<5> Disable DHCP for the physical function (PF), unless strictly required.
+<6> The number of SR-IOV virtual functions (VFs) to instantiate.
+<7> It is recommended to set this to `up`.
+<8> Disable IPv4 addressing for the VF attached to the bond.
+<9> A minimum transmission rate, in Mbps, for the VF. This sample value sets a rate of 100Mbps.
+    * This value must be less than or equal to the maximum transmission rate.
+    * Intel NICs do not support the `min-tx-rate` parameter. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1772847[*BZ#1772847*].
```

Comment:
I know it happens, but I don't think we usually like to link to BZs in the doc text. Just seeing if this is critical to mention here or not

Reply:
I think it is valid here and I was asked to add it plus it is also present here https://docs.openshift.com/container-platform/4.12/networking/hardware_networks/configuring-sriov-net-attach.html#configuring-sr-iov-networks_configuring-sriov-net-attach so hopefully OK to go with it